1.9/storage/persistent-volume.md

```diff
@@ -73,10 +71,10 @@ The second volume is a persistent volume with a `containerPath` that matches the
 
 For a complete example, see the [Running stateful MySQL on Marathon](#stateful-sql) section.
 
-## Creating a stateful application via the DC/OS Web Interface
+## Creating a stateful application via the DC/OS GUI
```

Comment:
Is `DC/OS GUI` the official term? People in engineering usually refer to it as the `DC/OS UI` (leaving out the G(raphical)) but I can't tell how this is handled in DC/OS docs.